tripal/src/TripalImporter/TripalImporterBase.php

```diff
@@ -584,6 +602,16 @@ protected function setItemsHandled($total_handled) {
   }
 
   /**
+   * Performs tasks after the importer has completed.
+   *
+   * @return void
+   *   No return value.
+   */
+  public function postRun() {
+
+  }
+
+    /**
```

Comment:
This somehow got mis-indented in this PR?

```suggestion
  /**
```